SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -233,7 +233,7 @@ bool GenerateLaunchConfig(const FString& LaunchConfigPath, const FSpatialLaunchC
 
 		// Populate json file for launch config
 		Writer->WriteObjectStart(); // Start of json
-			Writer->WriteValue(TEXT("template"), LaunchConfigDescription.Template); // Template section
+			Writer->WriteValue(TEXT("template"), LaunchConfigDescription.GetTemplate(LaunchConfigDescription.bUseDefaultTemplateForRuntimeVariant)); // Template section
```

Comment:
Can this be wrapped in a helper method ? The same object is referenced twice is the call.

Reply:
Nah, the `GetTemplate` is used with `true` later on the fetch the default template names based on runtime variant.

Reply:
I think Nic's saying is it'd be better to have two functions - ie. GetDefaultTemplateForRuntimeVariant, and whatever the other option is. Rather than having a function take a bool.

Reply:
This is actually how I had this initially setup. I didn't like it because of the duplicate code in the `GetTemplate` and `GetDefaultTemplateForRuntimeVariant` functions. Although just realised I can make `GetTemplate` call `GetDefaultTemplateForRuntimeVariant` if the var is set.